src/Lesson2/Task3.java

```diff
@@ -0,0 +1,24 @@
+package Lesson2;
+
+import java.util.Scanner;
+
+public class Task3 {
+    //Дано целое число. Если оно является положительным, то прибавить к нему 1.
+    // Если отрицательным, то вычесть из него 2.
+    // Если нулевым, то заменить его на 10.
+    // Вывести полученное число
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int i = sc.nextInt();
+        if (i == 0) {
+            i = 10;
+        } else if (i > 0) {
+            i += 1;
+        } else if (i < 0) {
+            i -= 2;
+        } else {
+            System.out.println("you input incorrect number");
```

Comment:
clarification: if you enter incorrect value, for example String, you get Exception, because nextInt() waits int value